webextensions/api/devtools.json

```
@@ -502,7 +502,7 @@
               "__compat": {
                 "support": {
                   "chrome": {
-                    "version_added": true
+                    "version_added": "19"
```

:x: Same as `inspectedWindow`

```suggestion
                    "version_added": "18"
```